app-text/aspell/files/aspell-0.60.6.1-unicode.patch

```diff
@@ -0,0 +1,299 @@
+--- Makefile.am
++++ Makefile.am
+@@ -121,7 +121,7 @@ EXTRA_DIST += common/*.hpp common/*.h\
+ # Aspell Program
+ #
+ 
+-AM_CPPFLAGS += -DLOCALEDIR="$(localedir)"
++AM_CPPFLAGS += -DLOCALEDIR="$(localedir)" $(NCURSES_CFLAGS)
+ 
+ bin_PROGRAMS = word-list-compress aspell prezip-bin
+ 
+@@ -129,7 +129,7 @@ word_list_compress_SOURCES = prog/compress.c
+ 
+ aspell_SOURCES = prog/aspell.cpp prog/check_funs.cpp prog/checker_string.cpp
+ 
+-aspell_LDADD = libaspell.la $(CURSES_LIB) $(LTLIBINTL)
++aspell_LDADD = libaspell.la $(NCURSES_LIBS) $(LTLIBINTL)
+ 
+ prezip_bin_SOURCES = prog/prezip.c
+ 
+--- configure.ac
++++ configure.ac
+@@ -53,13 +53,10 @@ AC_ARG_ENABLE(win32-relocatable,
+   [  --enable-win32-relocatable])
+ 
+ AC_ARG_ENABLE(curses,           
+-  AS_HELP_STRING([--enable-curses=LIBFILE],[cursor control library]))
++  AS_HELP_STRING([--enable-curses],[cursor control library]))
+ 
+-AC_ARG_ENABLE(curses-include,           
+-  [  --enable-curses-include=DIR])
+-
+-AC_ARG_ENABLE(wide-curses,
+-  AS_HELP_STRING([--disable-wide-curses],[disable wide char utf8 cursor control]))
++AC_ARG_ENABLE(unicode,
++  AS_HELP_STRING([--enable-unicode],[enable Unicode support]))
+ 
+ AC_ARG_ENABLE(regex,
+   [  --disable-regex])
+@@ -312,192 +309,83 @@ AC_TRY_LINK(
+   [AC_MSG_RESULT(no)]
+ )
+ 
+-AC_SUBST(CURSES_LIB)
+-AC_SUBST(CURSES_INCLUDE)
+-
+ if test "$enable_curses" != "no"
+ then
+-  use_curses=t
+-  case "$enable_curses" in
+-    yes | ""                     )                             ;;
+-    /* | *lib* | *.a | -l* | -L* ) CURSES_LIB="$enable_curses" ;;
+-    *                            ) CURSES_LIB=-l$enable_curses ;;
+-  esac
+-  case "$enable_curses_include" in
+-    yes | no | "")                                         ;;
+-    -I*          ) CURSES_INCLUDE="$enable_curses_include" ;;
+-    *            ) CURSES_INCLUDE=-I$enable_curses_include ;;
+-  esac
+-fi
+-
+-if test "$use_curses"
+-then
+-
+-  ORIG_LIBS="$LIBS"
+-  ORIG_CPPFLAGS="$CPPFLAGS"
+-  CPPFLAGS="$CURSES_INCLUDE $ORIG_CPPFLAGS"
+ 
+-  if test -z "$CURSES_LIB"
++  if test "$enable_unicode" != "no"
+   then
++    AC_DEFINE(HAVE_WIDE_CURSES, 1, [Defined if curses library includes wide character support])
```

Comment:
Thanks, I will correct it. I have never really worked with Autotools from the developer's side.